docs/settings.rst

```diff
@@ -100,6 +100,12 @@ and eras; you can list them using the widget's ``available_layers`` method::
      'VLSS: VLA Low-frequency Sky Survey (Radio)', 'WISE All Sky (Infrared)',
      'WMAP ILC 5-Year Cosmic Microwave Background']
 
+In the Jupyter version, it is possible to add GUI controls that allow the 
+layers to be chosen from drop down menus. The controls also include a slider 
```

Comment:
Maybe describe in the text here how to get it, before showing the screenshot. Something like, after 'menus':

``
To get these, type::

    wwt.layer_controls

The controls also include...
```